modules/react/tokens/index.ts

```diff
@@ -12,7 +12,14 @@ import {
   CanvasSpaceNumbers,
   spaceNumbers,
 } from './lib/space';
-import {type, CanvasType, fontFamily, monoFontFamily, CanvasTypeVariant} from './lib/type';
+import {type,
+  fontFamily,
+  monoFontFamily,
+  CanvasType,
+  CanvasTypeProperties,
+  CanvasTypeVariants,
+  CanvasTypeHierarchy,
+} from './lib/type';
 import {CSSProperties} from './lib/types';
 
 const {default: colors, gradients, ...semanticColors} = canvasColorsWeb;
```

Comment:
FYI: This `default: colors` is going to bite us one day with the conflict from `semanticColors.colors`